File: src/front/domains/compe/CompeResponse.ts
import Events from "../events/Events";

export default class CompeResponse {
    constructor(
        public compeId = "",
        public compeName = '',
        public compePlace = '',
        public compeDates = '',
        public compeGuidelinesFile = null,
        public compeEvent: Events[] = []
    ){}
}

File: src/main/java/com/rakuriku/rakuriku/presentation/controller/compe/response/CompeResponse.java
package com.rakuriku.rakuriku.presentation.controller.compe.response;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CompeResponse {
    private String compeId;
	private String compeName;
	private String compeDates;
	private String compePlace;
	private String compeFeeType;
	private Serializable compeEvent;
}
